### DISCUSSION TITLE
Is there any solution for anti drop sensor error 104?

### DISCUSSION THREAD

**Discussion by acrabby** [mrbungle64/ecovacs-deebot.js#141] [General]
Hi,
I could not find any solution to get rid of the drop sensor malfunction errors. Is there any method, parameter, etc. to override this blocking error? I can even stop to LDS but 6 silly IR diodes block my device.

Deebot Ozmo 950:
"DownSensorAbnormal: Excess dust on the Anti-Drop Sensors (104)"

**Comment by mrbungle64** [comment]
@ayengec 

I'm not sure if I can help you. It seems that you have a problem with your LDS ...
Do you want to suppress this error in general or only when using this library?

**Reply by acrabby** [reply]
Hi, no it does not related my LDS sensor. I only wanted to mention that I could take control all of the sensors without anti drop sensors. Anti drop sensors are completely InfraRed emitter diode + phototransistor. These are really cheap electronic components but these sensor problems/or motherboard based problem could not be controlled via Homeassistant, IOBroker, etc.
Actually I wanted to ask how can I disable these sensors?

**Reply by mrbungle64** [reply]
Unfortunately, I don't know of any way to disable these sensors.

**Reply by acrabby** [reply]
Thank you so much for interesting. Finally I need to examine why the robot is out of order. I will try to reverse engineering on its motherboard. If I can learn to fix this issue, I probably will inform you in here.
